setup.py

```diff
@@ -45,7 +45,7 @@ def get_version(path):
 PYTHON_REQUIRES = ">=3.6"
 PACKAGES = setuptools.find_packages()
 VERSION = get_version("slickml/__init__.py")
-INSTALL_REQUIRES = get_install_requirements("requirements.txt"),
+INSTALL_REQUIRES = (get_install_requirements("requirements.txt"),)
```

Comment:
I think "," in the code was wrong in the beginning. Please change it to this:

INSTALL_REQUIRES = get_install_requirements("requirements.txt")